sre-operational-fabric.md

```diff
@@ -0,0 +1,119 @@
+Science or equivalent experience. Usually requires 8+ years of information
+systems design/architecture/development experience. May require additional
+certifications depending on specialization.
+
+### Physical Demands
+
+General office environment. Considerable stress may occur at times. Some
+lifting up to 20 pounds required. General office environment.  May require long
+periods sitting and viewing a computer monitor.  Moderate levels of stress may
+occur at times.  No special physical demands required.
+
+## Focus
+
+As a Rackspace Private Cloud (RPC) Operational Fabric SRE, you will help
+support our organizational vision: To become the preferred brand for consuming
+complex open source infrastructure, as-a-Service, delivering industry-leading
+simplicity, efficiency, and reliability.
+
+Our team supports this mission by improving
+[observability](https://en.wikipedia.org/wiki/Observability) (i.e. insight
+into) and [controllability](https://en.wikipedia.org/wiki/Controllability)
+(i.e. state mutation) of cloud deployments located in datacenters throughout
+the world.
+
+### About Us
+
+- **Team Size**: Currently 8 senior developers/engineers and a technical manager
+- **Hours**: We have flexible working hours and aim for a median of 35-40 hours a week
+- **Location**: US/UK work from home with offices in US/UK (most states supported)
+- **Travel**: About once or twice a year for a conference or team gathering
+
+### Responsibilities
+
+- Help develop and maintain **observability**/controllability of customer environments
+- Help drive zero downtime (0DD) CI/CD for our team's infrastructure and app stack
+- Work directly with support engineers, operations, product management, and end-users
+- Develop automation to improve efficiency, reliability, and simplicity of clouds
+- Take high level problems and autonomously deliver solutions supported by data
+- Develop solutions that leverage and integrate existing open source technologies
+- Engage asynchronously with team members via Slack, Jira, github, and email
+- Engage synchronously with team members via video calls
+
+### Qualifications
+
+The ideal candidate will have the following (**not** hard requirements):
+
+- Experience defining and operating production systems that run at scale
+- Experience developing and deploying applications at scale
+- Experience working on distributed teams
+- Experience with 0DD CI/CD on Kubernetes using Helm
+- Experience with multiple cloud computing technologies (e.g. AWS, GCP, OpenStack)
+- Experience with multiple programming paradigms (i.e. declarative, functional, imperative, object-oriented)
+- A polyglot (e.g. Python, Go, Haskell)
+- Good working knowledge of network security and secure coding fundamentals (e.g. auditing, authentication, authorization, confidentiality, integrity, availability, input safety)
+- Good working knowledge of messaging technologies (e.g. RabbitMQ, Kafka)
```

Comment:
RabbitMQ and Kafka aren't comparable. That Kafka can be used like RabbitMQ is an accident. You could probably say `of messaging and distributed logs`, and preserve the parenthetical as-is.

If you really want to ring some bells, saying "event sourced architecture" next to Kafka will excite a few folks, and indicates that we understand what Kafka is and is for.

Reply:
changed